ref/feature/ldapRegistry/examples.adoc

```diff
@@ -37,12 +38,12 @@ The following example shows custom and group filters that are specified for the
 
 === LDAPS
 
-To enable a secure connection to the LDAP server for encrypted communication via TLS or SSL protocols, configure the `<ldapRegistry>` element for Lightweight Directory Access Protocol Over Secure Socket Links (LDAPS).
-Set the `sslEnabled` attribute to `true` and the `sslRef` attribute to your `ssl id` value. In order to enable a secure TLS or SSL connection, you must have these connections configured in the `ssl` element.
+To enable a secure connection to the LDAP server for encrypted communication via TLS or SSL protocols, configure the `<ldapRegistry>` element to enable Lightweight Directory Access Protocol Over Secure Socket Links (LDAPS).
```

Comment:
Here you have "`<ldapRegistry>` element", and later in the topic it's just "`ldapRegistry` element". Maybe make this consistent?